src/main/webapp/scheduled-interviews.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Scheduled Interviews</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="scheduled-interviews.css">
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+      integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+      integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script src="scheduled-interviews.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <!-- TODO: add navbar here. -->
+      <h1>Scheduled Interviews</h1>
+      <!-- TODO: Render all scheduled events using Datastore information and Javascript, remove hardcoded ones. -->
+      <section id="scheduled-interviews-cards">
+        <div class="row">
+          <div class="col-sm-6">
+            <div class="card" style="width: 18rem;">
+              <div class="card-body">
+                <h5 class="card-title">July 17, 2020</h5>
+                <p class="card-text">3:00PM - 4:00PM EST</p>
+              </div>
+              <ul class="list-group list-group-flush">
+                <li class="list-group-item">Your role: Interviewer</li>
```

Comment:
I think the user's role should be more prominent, probably at the top of the card with the same size as the date (<h5>). You may want to reformat it as "You are interviewing <first name of interviewee>" and "<first name of interviewer> is interviewing you," but the specifics of that are up to you. The main point is that it should be easy to tell at a glance what the user's role will be.